app/controllers/registrations/account_not_found_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Registrations::AccountNotFoundController < ApplicationController
+end
```

Comment:
How do empty controllers work exactly? I'm wondering out of my own curiosity,

Reply:
Convention; they render the view that corresponds to the action.